ui/src/translations/ru.json

```
@@ -0,0 +1,813 @@
+    "steps": "Шаги",
+    "state": "Состояние",
+    "search term in message": "Поиск по термину в сообщении",
+    "search": "Поиск",
+    "search blueprint": "Поиск blueprints",
+    "all tags": "Все теги",
+    "source search": "Поиск по источнику",
+    "filter by log level": "Фильтр по уровню log",
+    "selected": "Выбрано",
+    "task": "Task",
+    "task logs": "Task logs",
+    "display flow {id} executions": "Показать выполнения flow {id}",
+    "actions": "Действия",
+    "select datetime": "Выберите дату",
+    "invalid field": "Неверное поле: {name}",
+    "required field": "Обязательное поле",
+    "error": "Ошибка",
+    "form": "Форма",
+    "form error": "Ошибка формы",
+    "display topology for flow": "Показать топологию для flow",
+    "cannot create topology": "Не удалось создать топологию для flow",
+    "start date": "Дата начала",
+    "end date": "Дата окончания",
+    "creation": "Создание",
+    "flow creation": "Создание flow",
+    "start datetime": "Дата и время начала",
+    "end datetime": "Дата и время окончания",
+    "restart": "Перезапуск",
+    "restart latest revision": "Перезапустить последнюю редакцию",
+    "restart tooltip": "Перезапустить выполнение с <code>{state}</code> task",
+    "restarted": "Выполнение перезапущено",
+    "restart confirm": "Вы уверены, что хотите перезапустить выполнение <code>{id}</code>?",
+    "restart change revision": "Вы можете изменить редакцию, которая будет использоваться для нового выполнения.",
+    "replay": "Повтор",
+    "replay from task tooltip": "Создать аналогичное выполнение, начиная с task <code>{taskId}</code>",
+    "replay from beginning tooltip": "Создать аналогичное выполнение, начиная с начала",
+    "replay latest revision": "Повтор с использованием последней редакции",
+    "replayed": "Выполнение повторено",
+    "replay confirm": "Вы уверены, что хотите повторить это выполнение <code>{id}</code> и создать новое?",
+    "prefill inputs": "Предзаполнение",
+    "current": "текущий",
+    "change status": "Изменить статус",
+    "change status done": "Статус task был обновлен",
+    "change status confirm": "Вы уверены, что хотите изменить статус task <code>{task}</code> для выполнения <code>{id}</code>?",
+    "change status hint": {
+      "WARNING": "['Flow будет помечен как WARNING.', 'Следующие tasks будут выполнены.', 'Tasks с ошибками будут выполнены.']",
+      "FAILED": "['Flow будет помечен как FAILED.', 'Другие tasks не будут выполнены.', 'Tasks с ошибками будут выполнены.']",
+      "SUCCESS": "['Flow перезапустится, так как этот task был успешным.', 'Все заблокированные tasks будут выполнены.', 'Flow будет в состоянии SUCCESS, если все запуски tasks будут успешными.']",
+      "RUNNING": "['Flow перезапустится и выполнит все следующие tasks.', 'Все заблокированные tasks будут выполнены.']"
+    },
+    "mark as": "Пометить как <code>{status}</code>",
+    "kill": "Убить",
+    "kill parents and subflow": "Убить родителей и subflows",
```

```suggestion
    "kill parents and subflow": "Убить родителей и подпроцессы",
```

but this sounds a bit disturbing - maybe in community people will contribute to something better

---

yeah, we need to be careful about the wording here. German translation was initially quite brutal too so I had to soften it 

I think your "Kill parents and subprocesses" sounds great!